### ISSUE TITLE
Exit replaying mode?

### ISSUE THREAD

**Issue by nguyenhoanganhduy** [jevakallio/vscode-hacker-typer#20]
Hi @jevakallio ,

I really like the project, but there is one thing that's been bugging me. Is there a way to exit replaying mode? That would be nice to stop replaying without reloading the window though.

**Comment by jevakallio** [comment]
Yes, agreed! I currently don't have a huge amount of time to dedicate to this project, but if you'd like to contribute a PR with this command, I'd be more than happy to accept it!

**Comment by nicoespeon** [comment]
I think #22 fixes this issue.

@jevakallio is there a way I can help you with the currently open PRs? 😃
